projects/1.18/assets/applied-botanics-addon/applied-botanics/appbot/lang/zh_cn.json

```
@@ -0,0 +1,3 @@
+{
+  "item.appbot.mana_p2p_tunnel": "P2P通道-魔力"
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "item.appbot.mana_p2p_tunnel": "P2P通道 - 魔力"
```

---

完全参照1.18AE2译名